### PR TITLE
[bugfix] sidepane rerender contextual menu blip and rearrange menu item order

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What

Resolve unwarranted rerenders of the sidepane causing ALL contextual menus to blip any time a remote participant state changes.

Also rearrange menu item orders as per figma designs.

# Why

Using the const like so <ContentRender/> invokes the function/component elsewhere in the JSX tree resulting in an anonymous.component to be generated.
By calling/invoking the const like so {ContentRender.?()} will invoke the function immediately at runtime and not elsewhere in the JSX tree.

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx [hunk 1]
```diff
@@ -100,11 +100,11 @@ export const SidePane = (props: SidePaneProps): JSX.Element => {
     <Stack verticalFill grow styles={paneStyles} data-ui-id="SidePane" tokens={props.mobileView ? {} : sidePaneTokens}>
       {HeaderToRender}
       <Stack.Item verticalFill grow styles={paneBodyContainer}>
-        <Stack verticalFill styles={scrollableContainer}>
+        <Stack verticalFill styles={scrollableContainer} key="asdf1">
```
[the remark this answers is not in the document]
Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx:103]: Aww, I like my asdf1

[the remark this answers is not in the document]
Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx]: Removed!

[the remark this answers is not in the document]
Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx]: fyi, I removed the other instances of asdf. regardless of how heartbreaking it was to see them go. 🥲

File: packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx [hunk 2]
```diff
@@ -322,18 +324,20 @@ export const usePeoplePane = (props: {
 
   const onRenderContent = useCallback((): JSX.Element => {
     return (
-      <>
+      <div key="asdfusepeoplepane">
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx:327]: still some asdf's left (revert that this is a div too)

Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx]: My vscode was in a weird state where the commit with asdf removal did not work... It was very odd. 
I resolved it with a hard reset of vscode. Should be good now!

File: packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx [hunk 3]
```diff
@@ -103,9 +103,7 @@ export const SidePane = (props: SidePaneProps): JSX.Element => {
         <Stack verticalFill styles={scrollableContainer}>
           {ContentRender && (
             <Stack.Item verticalFill styles={scrollableContainerContents}>
-              <Stack styles={containerContextStyles}>
-                <ContentRender />
-              </Stack>
+              <Stack styles={containerContextStyles}>{ContentRender?.()}</Stack>
```
Comment (PorterNan) [inline packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx:106]: we should also rename this into a function style - contentRenderer

File: packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx [hunk 4]
```diff
@@ -117,9 +115,7 @@ export const SidePane = (props: SidePaneProps): JSX.Element => {
                   : scrollableContainerContents
               }
             >
-              <Stack styles={containerContextStyles}>
-                <OverrideContentRender />
-              </Stack>
+              <Stack styles={containerContextStyles}>{OverrideContentRender?.()}</Stack>
```
Comment (PorterNan) [inline packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx:118]: Same here camel case

**Review by PorterNan — APPROVED** [review]

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**